### ISSUE TITLE
Add a route for root url

### ISSUE THREAD

**Issue by CaptainDaVinci** [CaptainDaVinci/Music-DL-Server#1]
Right now the application has only one path, `/download`. It would be nice to have `/` and `/index/` path as well.

**Comment by jai-dewani** [comment]
I think we can solve this issue and simultaneously create a frontend for web users. 
I can create a simple index page that can take youtube URLs with encoding and quality as input and download the same for them.

**Comment by CaptainDaVinci** [comment]
Yes, that's the plan (#2 ) I created separate issues to help with getting started. But feel free to create  a PR which addresses both.

**Comment by CaptainDaVinci** [comment]
For UI, I don't really have anything in mind, but looking at a very simple interface (text field, 2 spinners maybe) for now.

**Comment by jai-dewani** [comment]
So Should I try to make an attempt at it?

**Comment by CaptainDaVinci** [comment]
Sure, go ahead! :)
